test/test_video_reader.py

```diff
@@ -282,45 +281,46 @@ def check_separate_decoding_result(self, tv_result, config):
         video_duration = vduration.item() * Fraction(
             vtimebase[0].item(), vtimebase[1].item()
         )
-        self.assertAlmostEqual(video_duration, config.duration, delta=0.5)
+        assert abs(video_duration-config.duration)<=0.5
```

Comment:
have you tried using `pytest.approx` here? https://docs.pytest.org/en/6.2.x/reference.html#pytest-approx

I think we should be able to use it everywhere `assertAlmostEqual` was used